index.js

```diff
@@ -4,7 +4,36 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    console.log(process.argv)
+    let operator = process.argv[2]
+    // checking for no operation
+    if (!operator){
+        return "No operation provided..."
+    } 
+    // checking for numbers
+    if (process.argv[3] === undefined){
+        return "No numbers provided..."
+    }
+    // check for other operators that are not plus or minus
+    if (!(operator.includes("plus") || operator.includes('minus'))) {
+        return `Invalid operation: ${operator}`
+    }
+    // for loop: initialize, condition, change
+    let sum = Number(process.argv[3])
```

Comment:
Since you can add or subtract to the number a more accurate variable name might be something like `total` or `result`